client/app/components/examples/dialog.vue

```diff
@@ -0,0 +1,104 @@
+<template lang="pug">
+    extends ../../templates/layout.pug
+
+    block content
+        include ../../templates/category_list.pug
+
+    block dialog
+        div(v-bind:aria-hidden="hidden", role="dialog")
+            div(role='separator', v-on:click="onClose")
```

Comment:
This one isn't a separator as wanted by the _ARIA_ spec. It's prpobably more a `backdrop` (as in the `<dialog/>` component) but we don't have any `backdrop` role. So I suggest to simply use a `.backdrop` class.